amgut/lib/data_access/ag_data_access.py

```diff
@@ -386,7 +386,8 @@ def deleteSample(self, barcode, ag_login_id):
             sql = """UPDATE ag_kit_barcodes
                      SET participant_name = NULL, site_sampled = NULL,
                          sample_time = NULL, sample_date = NULL,
-                         environment_sampled = NULL, notes = ''
+                         environment_sampled = NULL, notes = NULL,
```

Comment:
No, notes can be null, so this is fixing it so that it is consistent with a non-logged barcode instead of slightly different with the empty string.